gradle.properties

```diff
@@ -11,4 +11,4 @@
 #
 org.gradle.daemon=true
 org.gradle.configureondemand=true
-org.gradle.caching=true
+org.gradle.caching=false
```

Comment:
why?

Reply:
This is a bug that we are seeing in the dev environment. Due to caching being enabled, gradle doesnt seem to build  the modules where a new file is added, but the exiting files have no changes in them. I think it would be a good idea to fix this in a separate ticket as this is unrelated to FindToken changes.